server/tests/conftest.py

```
@@ -46,6 +46,20 @@ def teardown():
     return _db
 
 
+@pytest.fixture(scope="function")
+def db(app, request):
```

Ruff (0.13.1)

Redefinition of unused `db` from line 37

(F811)